demos/text-field.scss

```diff
@@ -28,7 +28,7 @@
   @include mdc-text-field-hover-bottom-line-color($hover-border);
   @include mdc-text-field-line-ripple-color($focused-border);
   @include mdc-text-field-ink-color(black);
-  @include mdc-text-field-label-color(rgba(blue, .5));
```

Comment:
My bad - I didn't realize this mixin already existed. Let's undo the rename since it would be a breaking change unrelated to this PR.